customer-onboarding-showcase/customer-onboarding-starter/customer-onboarding-starter-core/src/main/java/io/miragon/miranum/examples/customer/onboarding/adapter/in/rest/PlaceApplicationController.java

```diff
@@ -0,0 +1,38 @@
+package io.miragon.miranum.examples.customer.onboarding.adapter.in.rest;
+
+import io.miragon.miranum.examples.customer.onboarding.adapter.in.rest.dtos.PlaceApplicationRequestDto;
+import io.miragon.miranum.examples.customer.onboarding.adapter.in.rest.dtos.PlaceApplicationResponseDto;
+import io.miragon.miranum.examples.customer.onboarding.application.port.in.PlaceApplicationInCommand;
+import io.miragon.miranum.examples.customer.onboarding.application.port.in.PlaceApplicationUseCase;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/place-application")
+@AllArgsConstructor
+@CrossOrigin
+public class PlaceApplicationController {
+
+    private final PlaceApplicationUseCase placeApplicationUseCase;
+
+    @CrossOrigin
+    @PostMapping
+    public ResponseEntity<PlaceApplicationResponseDto> placeApplication(@RequestBody PlaceApplicationRequestDto placeApplicationRequestDto) {
+        var placeApplicationInCommand = new PlaceApplicationInCommand(
+                placeApplicationRequestDto.getName(),
+                placeApplicationRequestDto.getEmail(),
```

Comment:
maybe extract mapping into functions or class to make this a tiny bit more readable and to make the intention (more) clear

Reply:
Hm, I am on the fence here. To me it seems already pretty clear. Especially since this example is not that big. 
Imho we do not need to over engineer such a demo.